tests/unit/test_plotting/test_quick_plot.py

```diff
@@ -464,9 +464,13 @@ def test_plot_2plus1D_spm(self):
         pybamm.close_plots()
 
     def test_failure(self):
```

Comment:
Can you call it something a bit more explicit (e.g. `test_wrong_type` or something like this)?

Reply:
Done